backend/user_management/utils.py

```
@@ -198,6 +198,20 @@
             headers={"WWW-Authenticate": "Bearer"},
         )
         
+        
+
+
+
+# Authenticate user from DB
+def authenticate_user(db: Session, username: str, password: str):
```

Ruff (0.8.2)

Missing return type annotation for public function `authenticate_user`

(ANN201)